var/spack/repos/builtin/packages/duckdb/package.py

```diff
@@ -106,6 +107,8 @@ def setup_build_environment(self, env):
                 env.set(make_flag, "1")
             elif "~" + flag in self.spec:
                 env.set(make_flag, "0")
+        if self.spec.satisfies("@:0.10.2"):
```

Comment:
I don't think this check matches the one in the description.  You state the environment variable is available "_after_ 0.10.0".  (https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges)

Reply:
After but not including. It may have been clearer to say from 0.10.2 onwards but I'm not positive which method 0.10.1 needs but we don't have a version for that

Reply:
I had the syntax backwards throughout. Thank you for reviewing carefully.